src/workstation-gui/Gemfile.lock

```diff
@@ -206,7 +206,7 @@ GEM
     chef-telemetry (1.1.1)
       chef-config
       concurrent-ruby (~> 1.0)
-    chef-utils (18.0.185)
+    chef-utils (18.1.0)
       concurrent-ruby
     chef-vault (4.1.10)
```

Comment:
shouldn't this one also bumped to 4.1.11?

Reply:
with rebase, we'll automatically have it when this PR is merged.

Reply:
I see, thanks!